app/uk/gov/hmrc/taxenrolmentassignmentfrontend/logging/LoggingEvent.scala

```diff
@@ -142,7 +142,7 @@ object LoggingEvent {
   def logIncorrectUserType(
     credentialId: String,
     expectedUserType: List[AccountTypes.Value],
-    actualUserType: Option[AccountTypes.Value]
```

Comment:
I don't think this should have been changed (I expect it is a consequence of setting the redirect url to be PTA )